cypress/integration/1230-doi_minting/1234-owner_drafts_a_doi_for_a_draft_publication/draft_doi_for_draft_registration.js

```diff
@@ -0,0 +1,50 @@
+import { Before } from 'cypress-cucumber-preprocessor/steps';
+import { Given, When, Then, And } from 'cypress-cucumber-preprocessor/steps';
+import { USER_DRAFT_DOI } from '../../../support/constants';
+
+const filename = 'example.txt';
+
+Before(() => {
+  cy.login(USER_DRAFT_DOI);
+  cy.startRegistrationWithFile(filename);
+  cy.get('[data-testid=registration-file-start-button]').should('be.enabled');
+  cy.get('[data-testid=registration-file-start-button]').click({ force: true });

```

Comment:
burde ikke disse være en del av `startRegistrationWithFile`?
```suggestion

```

Reply:
Det er noen tester hvor man ikke trykker på start knappen, kommer til å legge til enten en ny funksjon i command.js eller lage et common step for dette.